src/cpp/shared/adventure/hero.cpp

```diff
@@ -156,6 +156,20 @@ void hero::TakeArtifact(int art) {
 	}
 }
 
+int hero::CountEmptyArtifactSlots() {
+
+
+	int amount = 0; //The counted amount of empty artifacts that the player has
```

Comment:
Use 2 spaces instead of 1 tab for indentation everywhere (You can configure visual studio so it places spaces when you press tab)